### DISCUSSION TITLE
Extract wanted text from pattern

### DISCUSSION THREAD

**Discussion by tchingtchan** [explosion/spaCy#5412] [Help: Coding & Implementations] [answered]
{"label": "IMMAT", "pattern": [{"IS_ALPHA": true,"LENGTH": 1},{ "ORTH": "comme"},{"IS_ALPHA":true},{"IS_ALPHA": true,"LENGTH": 1},{ "ORTH": "comme"},{"IS_ALPHA":true},{"IS_DIGIT":true , "LENGTH": 3},{"IS_ALPHA": true,"LENGTH": 1},{ "ORTH": "comme"},{"IS_ALPHA":true},{"IS_ALPHA": true,"LENGTH": 1}]}

This pattern allow me to extract text like this : 'a comme alain b comme bernanrd 123 c comme charles d_IMMAT']
i want to get just this form of data : "ab123cd_IMMAT" so i want to hide "comme"+ "is_ALPHA" from the returned data

## Your Environment

* Operating System: mac os
* Python Version Used: 3.6.5
* spaCy Version Used: 
* Environment Information:

**Comment by adrianeboyd** [comment, marked as answer]
If you're not using any operators/quantifiers, the token position in the match should correspond to the token position in the pattern, so the first `{ "ORTH": "comme"}` in the pattern should correspond to the second token in the matched span.

If the pattern does include operators, unfortunately the `Matcher` doesn't support capturing groups at this point (see #3275), so there's no easy way to align the pattern with the matched tokens.
